requirements.docs.txt

```diff
@@ -1,2 +1,3 @@
 Sphinx==5.3.0;python_version=="3.6"
 Sphinx==7.0.1;python_version>"3.6"
+werkzeug==2.3.8
```

Comment:
Why this specific version of Werkzeug for docs requirements?

Reply:
This one was failing before because of a missing werkzeug. You are right, we don't have to specify a version for it here though. Relaxed this to just list werkzeug here.